tests/js/server/aql/aql-memory-limit.js

```diff
@@ -242,13 +242,21 @@ function ahuacatlMemoryLimitGraphQueriesTestSuite () {
     
     testKShortestPaths : function () {
       const query = "WITH " + vn + " FOR p IN OUTBOUND K_SHORTEST_PATHS '" + vn + "/test0' TO '" + vn + "/test11' " + en + " RETURN p";
-      
-      let actual = AQL_EXECUTE(query, null, { memoryLimit: 5 * 1000 * 1000 }).json;
-      // no shortest path available
-      assertEqual(1024, actual.length);
-      
+
+      if (isCluster) {
+        // TODO [GraphRefactor]: Refactored variant uses now more memory. Check if we can improve here.
+        // [GraphRefactor] Note: Related to #GORDO-1361
```

Comment:
Another TODO

Reply:
Yes we do have a ticket for this. This is also mentioned in L248. I think we can keep this to be able to find the related test.

Reply:
To be entirely honest, these `TODO`s in code rarely have enough context and description and linger around until the end of time;

I'd rather have a ticket with a description and remove the comments.

Reply:
changed. removed those todos